snapcraft/internal/cache/_cache.py

```diff
@@ -33,13 +33,14 @@ def __init__(self):
     def cache(self):
         raise NotImplementedError
 
-    def prune(self):
+    def prune(self, revision=None):
```

Comment:
declared in base class like: *def prune(self, *args, **kwargs)*,
and in subclass it looks like: *def prune(self, keep_revision, *args)